go/store/prolly/commit_closure_test.go

```diff
@@ -122,19 +125,19 @@ func TestCommitClosure(t *testing.T) {
 		ccr, err := NewEmptyCommitClosure(ns)
 		require.NoError(t, err)
 		e = ccr.Editor()
-		err = e.Add(ctx, NewCommitClosureKey(ns.Pool(), 0, hash.Parse("00000000000000000000000000000000")))
+		err = e.Add(ctx, NewCommitClosureKey(ns.Pool(), 0, addr))
 		assert.NoError(t, err)
-		err = e.Add(ctx, NewCommitClosureKey(ns.Pool(), 1, hash.Parse("00000000000000000000000000000000")))
+		err = e.Add(ctx, NewCommitClosureKey(ns.Pool(), 1, addr))
 		assert.NoError(t, err)
-		err = e.Add(ctx, NewCommitClosureKey(ns.Pool(), 1, hash.Parse("00000000000000000000000000000001")))
+		err = e.Add(ctx, NewCommitClosureKey(ns.Pool(), 1, addr))
 		assert.NoError(t, err)
-		err = e.Add(ctx, NewCommitClosureKey(ns.Pool(), 2, hash.Parse("00000000000000000000000000000000")))
+		err = e.Add(ctx, NewCommitClosureKey(ns.Pool(), 2, addr))
 		assert.NoError(t, err)
 		ccr, err = e.Flush(ctx)
 		assert.NoError(t, err)
 		ccrc, err := ccr.Count()
 		require.NoError(t, err)
-		assert.Equal(t, 4, ccrc)
+		assert.Equal(t, 3, ccrc) // TODO(taylor): why did this change from 4?
```

Comment:
Is it okay that this count changed? Or is something wrong

Reply:
lines 127 and 129 had two different addresses before, now they both have `addr`